src/user-utilities.test.js

```diff
@@ -51,27 +51,7 @@ function testGetFullName(){
     }
 }
 
```

Comment:
We should find out what is causing the test to fail rather than just deleting it